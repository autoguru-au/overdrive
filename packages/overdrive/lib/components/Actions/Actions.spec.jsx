import { render } from '@testing-library/react';
import * as React from 'react';

import { Actions } from './index';

describe('<Actions />', () => {
	it('should not throw', () => {
		expect(() => render(<Actions />)).not.toThrow();
	});

	it('should match the snapshot', () => {
		expect(render(<Actions />).container.firstChild).toMatchSnapshot();
	});
});
