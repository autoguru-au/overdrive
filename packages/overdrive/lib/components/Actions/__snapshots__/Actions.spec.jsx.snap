// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Actions /> should match the snapshot 1`] = `
<div
  class="reset_block__6swfds3 useBoxStyles_display_flex__7o6k06ev useBoxStyles_width_none__7o6k06eh useBoxStyles_makeResponsiveStyle_key_bp__7o6k06ff useNegativeMarginLeft_makeResponsiveStyle_key_bp__18e9yxq8 useNegativeMarginTop_makeResponsiveStyle_key_bp__7pu7y48 Columns_align_stretch__wkgcjh2b Columns_wrapping_wrap__wkgcjh28"
/>
`;
